config/presets/authentication/fixed.json

```
@@ -0,0 +1,11 @@
+{
+  "@context": "https://linkedsoftwaredependencies.org/bundles/npm/@solid/community-server/^0.0.0/components/context.jsonld",
+  "@graph": [
+    {
+      "comment": "This extractor always sets the credentials to the fixed value.",
```

```suggestion
      "warning": "DO NOT USE IN PRODUCTION, ONLY FOR DEVELOPMENT, TESTING, OR DEBUGGING."
      "comment": "This extractor always sets the credentials to the fixed value.",
```

It not allowed, then we can prepend this to the comment.

Should be added to all classes in this situation.

---

It's not allowed since `comment` is a special predicate known by components.js. Would actually be interesting to have components.js support `warning` and have it emit warnings when such a file gets imported.